.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,85 @@
+name: e2e
+
+on: [pull_request]
+
+jobs:
+  privilege-escalation:
+    name: Privilege escalation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install curl
+        run: |
+          sudo apt -y update
+          sudo apt -y install curl
+
+      - name: Download and install minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+          sudo dpkg -i ./minikube_latest_amd64.deb
+
+      - name: Start minikube and wait until minikube's CoreDNS is available
+        run: |
+          minikube start --driver=docker
+          kubectl wait --for=condition=available deployment coredns -n kube-system
+
+      - name: Build SRO
+        run: minikube image build -t sro:local .
```

Comment:
Can't we use the Makefile build target and inject it to minikube?
I am thinking about future potential changes in the build process, in that case we will need to update the job as well.

Reply:
We could do `make local-iamge-build` and then `minikube iamge load`, but that would only work with Docker, not Podman. WDYT?

Reply:
> but that would only work with Docker

Why is that? it used the default `CONTAINER_COMMAND` which is `podman` and can also be set to `docker`.

Reply:
Minikube can only import images that would be stored in the local docker daemon, so we would have to export and import the image, which sounds cumbersome.